src/requirements/Farcaster/hooks/useFarcasterChannels.ts

```diff
@@ -1,23 +1,21 @@
+import { useFarcasterAPI } from "@/hooks/useFarcasterAPI"
+import { NEYNAR_BASE_URL } from "@/hooks/useFarcasterAPI/constants"
+import type { NeynarAPIClient } from "@neynar/nodejs-sdk"
```

Comment:
I have added the neynar sdk to include types. To prevent this next time, we could maybe use this. Although the update didn't seem like a small one.